src/datasets/arrow_dataset.py

```diff
@@ -461,6 +463,33 @@ def from_csv(
             path_or_paths, split=split, features=features, cache_dir=cache_dir, keep_in_memory=keep_in_memory, **kwargs
         ).read()
 
+    @staticmethod
+    def from_json(
+        path_or_paths: Union[PathLike, List[PathLike]],
+        split: Optional[NamedSplit] = None,
+        features: Optional[Features] = None,
+        cache_dir: str = None,
+        keep_in_memory: bool = False,
+        **kwargs,
+    ):
+        """Create Dataset from JSON Lines file(s).
```

Comment:
It works for jsonl files but also for json !
You just need to add the `field` parameter to `from_json` and `JsonDatasetReader` pass it to the `Json` dataset builder at initialization.

It allows to load squad-like json files for example.
The json looks like this:
```
{
    "data": [...],  # the actual data are in the "data" 
    ...
}
```

Reply:
The `field` parameter was implicitly passed in `kwargs`, but I can make it explicit. Also in the docstring ;)

What about the other JsonConfig parameters: `use_threads`, `block_size`, `newlines_in_values`?

Reply:
Yes right ! I didn't noticed that it was actually passed with the kwargs.
I think it's better to have it as a named argument directly since it's a very important one.

For the other parameters I think it's fine to have them in the kwargs